convert2rhel/utils.py

```diff
@@ -206,9 +213,8 @@ def log_traceback(debug):
     if debug:
         # Print the traceback to the user when debug option used
         loggerinst.debug(traceback_str)
-    else:
-        # Print the traceback to the log file in any way
-        loggerinst.file(traceback_str)
+    # Print the traceback to the log file in any way
+    loggerinst.file(traceback_str)
```

Comment:
Combined with the change to `CustomLogger.debug()`, won't this give you double messages in the logfile when `tool_opts.debug` is false but `debug` here is true?

When `loggerinst.debug()` is called, it checks `tool_opts.debug`, and if that's `False` now it will automatically log with `LOG_LEVEL_FILE.level` as a fallback. Then this (now-)unconditional `loggerinst.file(traceback_str)` will log the same message a second time at `LOG_LEVEL_FILE.level`, unless I'm not understanding the code properly.

Reply:
you're right. fixed